gulpfile.js

```diff
@@ -121,7 +122,7 @@ gulp.task('compile-tests', function () {
 
 gulp.task('default', function () {
     runSequence(
-        'watch',
-        $.shell.task('PATH=node_modules/.bin:$PATH electron .')
+        'clean',
+        ['typescript', 'sass', 'react']
```

Comment:
Nope, you can use the watch task by running: ``gulp watch``, It's better to set by default the build task.

At the moment I removed ``$.shell.task('PATH=node_modules/.bin:$PATH electron .')`` because this doesn't work on windows